### ISSUE TITLE
Explicitly list dependencies in the README

### ISSUE THREAD

**Issue by dcolli23** [UM-ARM-Lab/cdcpd#37]
The README's dependency list is somewhat ambiguous right now and has no indication that you need some UM ARM lab-specific forks of e.g. pyrosmsg.

It would help to explicitly list what versions of packages we're using as well as where we actually cloned them from.

Additionally, the install scripts need to be updated.

**Comment by lichenTHU** [comment]
Yes. As for pyrosmsg  pybind11-catkin  jsk  arc_utilities, the installation of all of these package need to be  showed in README, and the install scripts need to be updated.
